### PR TITLE
add review to the address

### PR DESCRIPTION
**Pull request by tranhuyhoangbka**

### CODE REVIEW

File: app/views/reviews/create.js.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$("#review_content").val("");
+$("#show-score").raty({
+  readOnly: true,
+  score: <%= @review.address.average_point %>
+});
+$("#show-score+span").text("(<%= pluralize @review.address.number_votes, 'Vote' %>)");
+$("#review-number").text("<%= pluralize @review.address.reviews.size, 'Comment' %>");
```
Comment (samnangcattor) [inline app/views/reviews/create.js.erb]: I80n

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -3,6 +3,17 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  after_action :store_location
+
+  def store_location
+    if params[:return].present?
+      session[:return] = params[:return]
```
Comment (samnangcattor) [inline app/controllers/application_controller.rb]: you should  write in short  :  session[:return] = params[:return]  if params[:return].present?

File: app/views/addresses/_review_area.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<header>
```
Comment (samnangcattor) [inline app/views/addresses/_review_area.html.erb]: Should be not  tag <header>  here , please remove or change to other  tag

File: app/controllers/reviews_controller.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+  before_action :set_address, :authenticate_user!
```
Comment (dieunb) [inline app/controllers/reviews_controller.rb]: don't use `set` or `get`. You can use `find` or load

**Pull request merged**